### ISSUE TITLE
Add empty alt="" to page scan images

### ISSUE THREAD

**Issue by erinchambers** [CDRH/data#253]
Right now, page scan images (that accompany TEI) do not have an alt attribute.

As explained in [WAI's alt text decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/), if the text in the image is also present as real text nearby, the image should have a) an _empty_ alt attribute and, for ARIA, b) a role of presentation. For example:

`<img src="page1.jpg" alt="" role="presentation">`

So we just need to find where the images are being transformed in the XSLT, and add the text string`alt="" role="presentation"` to the `img` element.
